devenv.nix

```diff
@@ -0,0 +1,41 @@
+{ pkgs, ... }:
+
+{
+  # https://devenv.sh/packages/
+  packages = [
+    pkgs.nodejs-18_x
+    pkgs.postgresql_14
+    pkgs.yarn
+  ];
+
+  # https://devenv.sh/processes/
+  processes = {
+    yarn-build-watch.exec = "yarn build:watch";
```

Comment:
Why are we running both these processes? It _should_ be enough to run just `yarn start:dev`, unless you've identified something I haven't?